src/Umbraco.Web/Runtime/WebInitialComponent.cs

```diff
@@ -131,7 +131,7 @@ private void SetupMvcAndWebApi()
             RouteNoContentController(umbracoPath);
 
             // register install routes
-            RouteTable.Routes.RegisterArea<UmbracoInstallArea>();
+           // RouteTable.Routes.RegisterArea<UmbracoInstallArea>();
```

Comment:
I assume we're just commenting this out because with these changes installation wont run in netframework?

Reply:
Yes, the current implementation cannot run in netframework. There controllers and everything is moved.